src/command.rs

```diff
@@ -70,9 +72,10 @@ pub async fn init(kind: PackageType, name: Option<PackageName>) -> miette::Resul
 
     manifest.write().await?;
 
-    PackageStore::create()
+    PackageStore::create(std::env::current_dir().unwrap_or_else(|_| ".".into()))
```

Comment:
Can we change the signature so people dont have to do this?

like `PackageStore::current().try_create()` or something like this?

Reply:
I thought we wanted to aim to be functional?